src/Nova/DomesticAddress.php

```diff
@@ -54,9 +54,9 @@ public function fields(Request $request)
             GoogleAutocomplete::make('Address')
                 ->countries('US')
                 ->withValues(['street_number.long_name','locality.long_name','postal_code.short_name','administrative_area_level_1.short_name'])->onlyOnForms(),
-            AddressMetadata::make(['Address Line 1'])->fromValue('street_number')->onlyOnForms(),
-            AddressMetadata::make(['City'])->fromValue('locality')->onlyOnForms(),
-            AddressMetadata::make(['Zip'])->fromValue('postal_code')->onlyOnForms(),
+            AddressMetadata::make('Address Line 1')->fromValue('street_number')->onlyOnForms(),
+            AddressMetadata::make('City')->fromValue('locality')->onlyOnForms(),
+            AddressMetadata::make('Zip')->fromValue('postal_code')->onlyOnForms(),
```

Comment:
@drewroberts this fix for the incorrect method call will fix the tests, but I don't know if the control will actually work.  With the original code, there's no way this form was ever called up and tested manually.

Reply:
Thank you. I'll have them do some manual tests on it today.

Reply:
CC: @joshtorres & @tipoff/omnia